frontend/src/types/media.ts

```diff
@@ -35,50 +36,133 @@ export interface Media {
    * - handle empty titles
    */
   title: string
-  /** the raw name of the creative work, as returned by the API */
+
+  /** The raw name of the creative work, as returned by the API. */
```

Comment:
Minor nit/personal preference: This isn't a sentence so it shouldn't start with an uppercase letter and should not end with a fullstop. I'm basing this on how dictionaries present definitions.